chrome/content/zotero/xpcom/translation/translate.js

```diff
@@ -169,13 +173,14 @@ Zotero.Translate.Sandbox = {
 					var note = notes[j];
 					if(!note) {
 						notes.splice(j--, 1);
-					} else if(typeof(note) == "object") {
-						// Canonicalize tags
-						if(note.tags !== undefined) note.tags = translate._cleanTags(note.tags);
-					} else {
+					} else if(typeof(note) != "object") {
 						// Convert to object
 						notes[j] = {"note":note.toString()}
 					}
+					// If we're not in a child translator canonicalize tags
```

Comment:
Here and above, add comma: "If we're not in a child translator, canonicalize tags"